Formula/v8.rb

```diff
@@ -114,6 +114,10 @@ def install
       treat_warnings_as_errors:     false, # ignore not yet supported clang argument warnings
     }
 
+    on_macos do
+      gn_args[:use_lld] = false # fixes https://github.com/Homebrew/homebrew-core/pull/84351#issuecomment-909621336
```

Comment:
We could probably actually just add this unconditionally, since we don't even use LLVM to compile on Linux.